site/content/docs/howto/supervisor/configure-supervisor-federationdomain-idps.md

```diff
@@ -0,0 +1,396 @@
+---
+title: Configure Identity Providers (IDPs) on a FederationDomain
+description: Learn how to use one or more identity providers, and identity transformations and policies, on a FederationDomain.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: IDPs on FederationDomains
```

Comment:
How about:
- `Multiple IDPs`
- `Weight 100`

Simple link, last in the list, after all of the `With <single IDP>` so its fairly obvious that its a different config?

Reply:
The new doc is about how to configure any IDPs going forward... one or many, not just how to configure multiple. We've been calling the feature "multiple IDPs" as a shorthand, but really it is much bigger than that. I'd rather avoid naming it anything about "multiple".

Reply:
<img width="254" alt="Screenshot 2023-09-12 at 1 37 15 PM" src="https://github.com/vmware-tanzu/pinniped/assets/280512/72c35584-3a5e-40c3-9594-2365fa170e11">

Reply:
- `Required Config`
- `FederationDomain Config`
?

Reply:
Let's consider reorganizing this new page and the pre-existing "Required Configuration" page in a future PR. For this PR, I just want to get the information about how to use the new features onto the site.